velox/type/Variant.cpp

```diff
@@ -555,7 +555,14 @@ uint64_t variant::hash() const {
       return folly::Hash{}(
           timestampValue.getSeconds(), timestampValue.getNanos());
     }
-
+    case TypeKind::SHORT_DECIMAL: {
```

Comment:
Should we also extend the `isNull` check above for decimal values?

Reply:
Are you suggesting to combine the cases for short and long and check for NULL here?